hooks/test_results.js

```diff
@@ -1,14 +1,72 @@
 const { event } = require('codeceptjs');
 const request = require('request');
+const Influx = require('influx');
+// const stringify = require('json-stringify-safe');
+
+const influx = new Influx.InfluxDB({
+  host: 'influxdb',
+  database: 'ci_metrics',
+});
 
 module.exports = function () {
-  event.dispatcher.on(event.test.finished, (test) => {
+  event.dispatcher.on(event.test.finished, async (test) => {
     console.log('********');
     console.log(`TEST: ${test.title}`);
     console.log(`RESULT: ${test.state}`);
     console.log(`RETRIES: ${test.retryNum}`);
     console.log(`TIMESTAMP: ${new Date()}`);
     console.log('********');
+
+    // console.log(stringify(test));
```

Comment:
I guess we can remove this.

Reply:
Leaving for now since we need it for getting some more data. Will remove once we finalize the metrics